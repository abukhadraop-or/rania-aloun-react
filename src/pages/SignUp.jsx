import React from 'react';
import Header from 'components/Header';

/**
 * Component holding temporary text for sign up page, to be implemented
 */
function SignUp() {
  return (
    <>
      <Header />
      <div>
        <p>ToDo _ Sign up page</p>
      </div>
    </>
  );
}

export default SignUp;
