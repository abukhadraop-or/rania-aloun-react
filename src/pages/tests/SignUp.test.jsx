import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SignUp from 'pages/SignUp';

describe('<SignUp />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<SignUp />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
