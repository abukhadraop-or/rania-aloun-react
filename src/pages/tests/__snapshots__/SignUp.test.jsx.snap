// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUp /> matches snapshot 1`] = `
<Fragment>
  <Header />
  <styled.div>
    <styled.div>
      <styled.h1>
        Sign Up
      </styled.h1>
      <styled.a
        href="www.google.com"
      >
        Have an account?
      </styled.a>
    </styled.div>
    <styled.form>
      <styled.input
        id="username"
        name="username"
        placeholder="Username"
        type="text"
      />
      <br />
      <styled.input
        id="email"
        name="email"
        placeholder="Email"
        type="text"
      />
      <br />
      <styled.input
        id="password"
        name="password"
        placeholder="Password"
        type="password"
      />
      <br />
      <styled.input
        type="submit"
        value="Sign Up"
      />
    </styled.form>
  </styled.div>
</Fragment>
`;
